stubs/geopandas/geopandas/explore.pyi

```diff
@@ -0,0 +1,63 @@
+from collections.abc import Callable, Hashable, MutableMapping, Sequence
+from typing import Any
+
+import branca  # type: ignore[import-not-found] # pyright: ignore[reportMissingImports]
+import folium  # type: ignore[import-not-found] # pyright: ignore[reportMissingImports]
+import pandas as pd
+import xyzservices  # type: ignore[import-not-found] # pyright: ignore[reportMissingImports]
+from matplotlib.colors import Colormap  # type: ignore[import-not-found]
```

Comment:
We usually declared a type alias for this, e.g.:

```python
Colormap: TypeAlias = Any  # from matplotlib.colors
```
But I'm actually interested how well this approach works.

Reply:
It works very well in this case because these imports usage is localized to places where the optional dependency would be present at runtime, so there is no confusion for type checkers.